src/pages/upload/index.tsx

```diff
@@ -74,18 +121,38 @@ const Upload: React.FC = () => {
         onError={(message) => console.log(message)}
         onFinish={(signResult, file) => {
           const fileUrl = signResult.signedUrl.split('?')[0]
-          console.log(fileUrl, signResult.publicUrl, file)
+          setLessonMetadata((prevState) => [
+            ...prevState,
+            {
+              title: file.name,
+              fileMetadata: {fileName: file.name, signedUrl: fileUrl},
+            },
+          ])
         }}
       />{' '}
       {state.files.map((file) => {
```

Comment:
This renders loading progress of individual uploads. Once loaded they disappear and the form fields are displayed instead